index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Angel Acosta</title>
+    <link rel= "stylesheet" href= "style.css">
+</head>
+<nav> 
+    <ul class="topnav"> 
+        <li><a id="top" href= "#intro">Interests</a></li>
+        <li><a id="top" href= "#work">Work</a></li>
+        <li><a class= "active" href="#start">Home</a></li>
+        <li><a id="top" href= "#projects">Projects</a></li>
+        <li><a id="top" href= "#contact">Contact</a></li>
+    </ul>    
+</nav>
+<body>
+
+    <div id = "start">
+        <h1>Hi 👋🏼  I'm Angel. Welcome!</h1>
+        <button><a href= "#intro">Read More About Me!</a></button> 
+    </div>
+    <div id="intro">
+        <hr>
+        <p>I'm learning how to 
+            <span><a href= "https://github.com/acostaa1" >build stuff</a></span>, 
+            how to <span><a href="https://www.masterclass.com/classes/daniel-negreanu-teaches-poker">take your money</a></span>, 
+            and how to <span><a href= "https://planitnerd.com">travel</a></span> for free. 
+            Also, I'm passionate about paying it forward and obssessed with all things Japan!</p>
+        <br>
```

Comment:
we typically like to avoid using `<br>`. that's what css is for!
we like to limit repeating code as much as possible and with `<br>`, you'd need to repeat a lot of those tags
also, what if you need to increase the margin between all of your sections? you'd need to add `<br>` EVERYWHERE
if you just used css, you can just simply increase the margin (´・◡・｀) 

like: 

```
<p>some paragraph</p>
<br>
<p>some paragraph</p>
<br>
<p>some paragraph</p>

```

those `<br>` are so ugly!
alternative: 
```
p {
  margin-bottom: 10px;
}
.....
<p>some paragraph</p>
<p>some paragraph</p>
<p>some paragraph</p>
```